### ISSUE TITLE
Cannot apply custom styles to `fluid` Container

### ISSUE THREAD

**Issue by monkishtypist** [sealninja/react-grid-system#135]
_TL;DR:_ Container component with `fluid` attribute does not accept custom styles with `style={{ }}`

**Goal:** to create a Container, Rows, and Cols with no padding for a flush image grid effect.

**Tried:** creating a Container and Row with the following:
```
<Container fluid style={{ padding: 0 }}>
  <Row nogutter>
    <Col>...</Col>
  </Row>
</Container>
```

**Actual Result:** styles were not applied to the Container resulting in left/right padding of half gutter width. No padding or negative margins in Row and Col worked as expected from `nogutter`, but this caused whitespace around Row from Container padding. This also means you cannot set a background color for example, or any other styles, using `style={{ }}` on the fluid Container.

**Expected Result:** the `style={{ }}` should be applied as normal and overwrite any associated inline styles.

**Debugging:** when using `style={{ }}` on a Container that _does not_ include the `fluid` attribute, everything works and styles are applied to the Container as normal.
```
<Container style={{ paddingLeft: 0, paddingRight: 0, maxWidth: '100%' }}>
  <Row nogutter>
    <Col>...</Col>
  </Row>
</Container>
```

**Work Around:** Method 1: removing the `nogutter` attribute from the Row and including `style={{ paddingLeft: 0, paddingRight: 0 }}` with every Col works for the un-padded grid effect. However you are still unable to include any other styles on the Container itself. Method 2: as _debugging_ section above, use a regular Container but apply inline styles to make it fluid.

**Suggested Fix:** apply styles to `<Container fluid>` same as on `<Container>`

**Comment by gerbenmeyer** [comment]
Nice catch, made  a fix for it!
